### ISSUE TITLE
[1.3.3-rc.0] Multiple exports with * not working

### ISSUE THREAD

**Issue by clayne11** [meteor/meteor#7167]
In `1.3.3-rc.0` if I have the following:

```javascript
// file1
export const testExport = 'testExport'
export const foo = 'foo'

// file2
export const test = 'test'

// /imports/package1/index.js
export * from './file1'
export {test} from './file2'

// /imports/package2/index.js
import {foo, test} from '/imports/package1'

console.log(foo) // undefined
console.log(test) // "test"
```

If I switch the order of the exports in `/imports/package1/index.js`, then this happens:

```javascript
// /imports/package1/index.js
export {test} from './file2'
export * from './file1'

// /imports/package2/index.js
import {foo, test} from '/imports/package1'

console.log(foo) // "foo"
console.log(test) // undefined
```

**Comment by benjamn** [comment]
If you have the time, do you think you could put together a failing test over here? https://github.com/benjamn/reify/blob/master/test/tests.js

**Comment by clayne11** [comment]
I'll try to find some time this weekend.

**Comment by clayne11** [comment]
Create a failing test and posted an issue in the other repo.

**Comment by benjamn** [comment]
This will be fixed in Meteor 1.3.3, which you can try now by running `meteor update --release 1.3.3`.
